app/services/send_eoc_second_deadline_reminder_email_to_candidate.rb

```diff
@@ -0,0 +1,6 @@
+class SendEocSecondDeadlineReminderEmailToCandidate
+  def self.call(application_form:)
+    CandidateMailer.eoc_deadline_reminder(application_form)
+    ChaserSent.create!(chased: application_form, chaser_type: :second_eoc_deadline_reminder)
+  end
+end
```

Comment:
Could we just have a single service class for both reminders that also accepts the chaser_type as an arg?